python/tests/test_collections.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import Mock
+
+from numerous import collection
+
+ORGANIZATION_ID = "test_org"
+COLLECTION_NAME = "test_collection"
+NESTED_COLLECTION_NAME = "nested_test_collection"
+COLLECTION_KEY = Mock(key="test_key", id="test_id")
+NESTED_COLLECTION_KEY = Mock(key="nested_test_key", id="nested_test_id")
+
+
+def test_collection()->None:
+    mock_client = Mock()
```

Comment:
Try to make a test where instead of mocking `NumerousClient`, you mock the graphql client that is uses instead